scripts/pylib/twister/twisterlib/testinstance.py

```diff
@@ -67,6 +67,8 @@ def __init__(self, testsuite, platform, outdir):
             self.build_dir = os.path.join(outdir, platform.normalized_name, source_dir_rel, testsuite.name)
         self.run_id = self._get_run_id()
         self.domains = None
+        # Instance need to use sysbuild if a given suite or a platform requires it
+        self.sysbuild = testsuite.sysbuild or platform.sysbuild
```

Comment:
what about scenario, when someone do not wat to use sysbuild and add to testcase.yaml `sysbuild: false`
see: https://github.com/nrfconnect/sdk-nrf/pull/15802

Reply:
for now it is ok, in PR I mentioned, there is nrf53 used, which does not have sysbuild set in platform.yaml. But good to provide (maybe not in scope of this PR) some solution to unset sysbuild in testcase.yaml

Reply:
We will think about it. Right now there are no use cases for this, as you've noticed. When there'll be a clear use case, we will add decide on certain ruling about a precedence of such configs defined at different layers